mixins/itemHelper.ts

```diff
@@ -116,6 +119,40 @@ const itemHelper = Vue.extend({
         name: routeName,
         params: routeParams
       }).href;
+    },
+    /**
+     * Returns the appropiate material design icon for the BaseItemDto provided
+     *
+     * @param {BaseItemDto} item - The item we want to get the icon for
+     * @returns {string} - The string that references the icon
+     */
+    getItemIcon(item: BaseItemDto): string {
+      switch (item.Type) {
+        case 'Audio':
+          return 'mdi-music-note';
+        case 'Book':
+          return 'mdi-book-open-page-variant';
+        case 'BoxSet':
+          return 'mdi-folder-multiple';
+        case 'Folder':
+        case 'CollectionFolder':
+          return 'mdi-folder';
+        case 'Movie':
+          return 'mdi-filmstrip';
+        case 'MusicAlbum':
+          return 'mdi-album';
+        case 'MusicArtist':
+        case 'Person':
+          return 'mdi-account';
+        case 'PhotoAlbum':
+          return 'mdi-image-multiple';
+        case 'Playlist':
+          return 'mdi-playlist-play';
+        case 'Series':
```

Comment:
Is there an icon we can use for episodes?

Reply:
``card-outline`` maybe? We can use that also as the default.

``television-box`` ``television-classic`` ``television-guide`` and ``television-shimmer`` are also good imo.

Reply:
I'm a fan of just `television` or `television-shimmer`, up to you.

Reply:
@Camc314 What about using the same we have for Series or you're looking for something different?

Reply:
Yeah that's fine, I can't see any other icons that would be good to distinguish between the two

Reply:
Actually probably best to keep them the same?

Reply:
@Camc314 To be honest I think it's the best because it's the one that makes more sense and keeps the consistency across all the other icons we use.